client/src/main/java/com/alibaba/nacos/client/config/filter/impl/ConfigMetadata.java

```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
  */
 
-package com.alibaba.nacos.client.config.common;
+package com.alibaba.nacos.client.config.filter.impl;
 
 /**
- * config common constants.
+ * There are some constants of the config request and response in the package scope .
  *
- * @author Nacos
+ * @author onewe
```

Comment:
you should add author in a new line, not replace the old author.

Reply:
I'm sorry about that. IDEA is automatic edit. I will be careful。